File: dev-docs/snippets/documents/get-root.swift
import JsBaoClient

// Fetch metadata for the app's shared root document. Swift returns an untyped
// `[String: Any]`.
func getRoot(client: JsBaoClient) async throws {
  // #region example
  let info = try await client.documents.getRoot()
  // #endregion example
  _ = info
}

File: dev-docs/snippets/documents/get-root.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Fetch metadata for the app's shared root document.
export async function getRoot(client: JsBaoClient) {
  // #region example
  const info = await client.documents.getRoot();
  // #endregion example
  return info;
}
